lerna.json

```diff
@@ -1,6 +1,5 @@
 {
   "$schema": "node_modules/lerna/schemas/lerna-schema.json",
-  "useWorkspaces": true,
```

Comment:
Was that not adding any value?

Reply:
my global lerna is on the latest which has this deprecated